### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by vaughnwalters**
MVP

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      <div class="col-xl-12">
+        <div class="attackOutput text-center">
+          <button id="attackBtn" class="btn btn-lg btn-primary btn-danger" disabled="disabled">ATTACK!</button>
+          <div id="outputDiv"></div>
+        </div>
+      </div>  
+    </div>
+  </div>  
+
+  <script src="/node_modules/jquery/jquery.min.js" type="text/javascript"></script>
```
Comment (morecallan) [inline index.html:75]: This script tag didn't work for me. If it's working for you with the way that node has it installed in your project files, we should talk to Joe and see what's up.

**Comment by morecallan** [comment]
Very nice! Big fan of your file structure and your commit hunks/messages. It's clear you have a strong grasp on best practices with modularization and organizing your code, and clearly commenting it. You also have a strong grasp on UX with only providing your users a singular path (you earn a gold UX star from me). You are nearly at MVP... but I want to stress just 2 things:

Prototypal inheritance: If you look at your prototypal chain, all of your functionality is in your Models. They aren't really inheriting any functionality from their parents in the chain. In order to really demonstrate knowledge of prototypal inheritance there are a few things you can do, like giving functions to the base Robot constructor that accept arguments (like say... calculating health or damage), things further down the chain can then pass arguments to those functions. You can also add bonuses at certain tiers. What you don't want are essentially worthless links in the chain (ones that just list the type name). I'm positive you'll be able to come up with something to improve this.

DRY. I know the DOM manipulation stuff is picky with the player construction but the fact that you have the same functionality twice should be a red flag. I won't require you to do this for MVP but rather challenge you to see if you can come up with a way of DRYing this up.

**Comment by vaughnwalters** [comment]
hey hey callan -
1. i pushed up a new commit on the mjd quiz branch that has some added functionality and inheritance in my prototypal chain based on adding more randomly generated health and attack at every level of the chain.  
2. did you get the script tag working?  it works fine in my project - loading fine when i inspect the network tab in dev tools.
3.  DRYing the code up.  I’ll work on this more tonight when i get home after the library kicks me out,  but I should be at MVP now with the added functionality of the prototypal chain.  

Thanks!
.vaughn

**Comment by morecallan** [comment]
👍 Lookin' good. I'm thinking the script tag might be an issue with the way NPM installs jquery. Keep this in mind for any projects you might be sharing with potential employers or other dev friends; you might need to go with CDN. Keep playing with this one as much as you have the patience for - you have a very strong foundation to play with it.

**Comment by vaughnwalters** [comment]
cool.  i’ll merge, but i have a couple of questions about it that i’ll ask you later today.

thanks callan

**Pull request merged**